src/pages/index.tsx

```diff
@@ -369,197 +283,137 @@ const HomePage = ({
-              headerKey="page-index:page-index-developers-code-examples"
-              isCode
-              border="1px"
-              borderColor="text"
-              boxShadow={cardBoxShadow}
-              maxWidth={{ lg: "624px" }}
-              ms={{ lg: 16 }}
-            />
-          </Box>
-          <FeatureContent>
-            <SectionHeading>
-              <Translation id="page-index:page-index-developers" />
-            </SectionHeading>
-            <SectionDecription>
-              <Translation id="page-index:page-index-developers-description" />
-            </SectionDecription>
-            <ButtonLinkRow>
-              <ButtonLink href="/developers/">
-                <Translation id="page-index:page-index-developers-button" />
-              </ButtonLink>
-            </ButtonLinkRow>
-          </FeatureContent>
-          {/* Render CodeModal & Codeblock conditionally */}
+              </div>
+            </div>
+          </div>
+        </HomeSection>
+
+        {/* TODO: Add "The Internet Is Changing" section */}
+
+        <HomeSection
+          tag="Builders"
+          title="Blockchain's biggest builder community"
+          imgSrc={buildersImage}
+        >
+          <p className="text-lg">
+            Ethereum is home to Web3’s largest and most vibrant developer
+            ecosystem. Use JavaScript and Python, or learn a smart contract
+            language like Solidity or Vyper to write your own app.
+          </p>
+          <div className="flex flex-wrap gap-6 py-8">
+            <ButtonLink
+              href="/developers/"
+              className={cn(
+                "w-fit no-underline",
+                buttonSizeLg,
+                buttonVariantSolid
+              )}
+            >
+              Builder&apos;s Portal <MdChevronRight />
+            </ButtonLink>
+            <ButtonLink
+              href="/developers/docs/"
+              className={cn(
+                "w-fit no-underline",
+                buttonSizeLg,
+                buttonVariantOutline
+              )}
+            >
+              Documentation
+            </ButtonLink>
+          </div>
+          <WindowBox
+            title={t("page-index:page-index-developers-code-examples")}
+            Svg={AngleBrackets}
+          >
+            {codeExamples.map(({ title, description }, idx) => (
+              <button
+                key={title}
+                className="flex flex-col space-y-0.5 border-t px-6 py-4 hover:bg-black/5"
+                onClick={() => toggleCodeExample(idx)}
+              >
+                <p className="font-bold">{title}</p>
+                <p className="text-start text-sm text-body-medium">
+                  {description}
+                </p>
+              </button>
+            ))}
+          </WindowBox>
+
           {isModalOpen && (
+            // TODO: Migrate CodeModal, CodeBlock, Skeleton from Chakra-UI to tailwind/shad-cn
```

Comment:
👍🏼 Got it, do you think you will cover those? or should I add them as part of the shadcn migration epic?

Reply:
Separate task from the homepage epic; would include in shadcn migration